contracts/token/ERC777/ERC777.sol

```diff
@@ -121,8 +130,18 @@ contract ERC777 is IERC777, IERC20 {
      * @param value uint256 the amount of tokens to be transferred
      */
     function transferFrom(address from, address to, uint256 value) external returns (bool) {
-        _transfer(msg.sender, from, to, value);
-        _approve(from, msg.sender, _allowances[from][msg.sender].sub(value));
+        require(to != address(0), "ERC777: transfer to the zero address");
+        require(from != address(0), "ERC777: transfer from the zero address");
+
+        address operator = msg.sender;
+
+        _callTokensToSend(operator, from, to, value, "", "");
+
+        _move(operator, from, to, value, "", "");
+        _approve(from, operator, _allowances[from][operator].sub(value));
+
+        _callTokensReceived(from, from, to, value, "", "", false);
```

Comment:
Is this a bug? :butterfly::sparkles: 

```suggestion
        _callTokensReceived(operator, from, to, value, "", "", false);
```

Reply:
It is! The sender and receipt tests only test `send` and `operatorSend`, so the test suite missed this.